runtime/compiler/control/CompilationThread.cpp

```diff
@@ -1662,7 +1662,7 @@ void TR::CompilationInfo::invalidateRequestsForUnloadedMethods(TR_OpaqueClassBlo
       TR_ASSERT(curCompThreadInfoPT, "a thread's compinfo is missing\n");
 
       TR_MethodToBeCompiled *methodBeingCompiled = curCompThreadInfoPT->getMethodBeingCompiled();
-      // Mark the method beign compiled that it has been unloaded.
+      // Mark the method begin compiled that it has been unloaded.
```

Comment:
I think 'being' was intended.

Reply:
Note: I should add `* being` to the commit message